rts/Rendering/Models/AssParser.cpp

```diff
@@ -510,16 +510,29 @@ void CAssParser::LoadPieceGeometry(SAssPiece* piece, const S3DModel* model, cons
 			// vertex normal
 			const aiVector3D& aiNormal = mesh->mNormals[vertexIndex];
 
-			if (!IS_QNAN(aiNormal))
+			if (IS_QNAN(aiNormal)) {
+				LOG_SL(LOG_SECTION_PIECE, L_DEBUG, "Malformed normal (model->name=\"%s\" piece->name=\"%s\" vertexIndex=%d x=%f y=%f z=%f)", model->name.c_str(), piece->name.c_str(), vertexIndex, aiNormal.x, aiNormal.y, aiNormal.z);
```

Comment:
Let's add some sane normal here, for example `vertex.normal = float3{0.0f, 1.0f, 0.0f}`